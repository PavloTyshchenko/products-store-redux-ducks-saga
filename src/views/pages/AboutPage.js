import React from 'react'
import { Jumbotron, Button } from 'react-bootstrap';

const AboutPage = props => {
    return (
        <Jumbotron>
            <h1>About</h1>
            <p>
                This is a simple react application to test myself,
                practice skills and obtain some new knowledge
            </p>
            <p>
                <Button variant="primary">Learn more</Button>
            </p>
        </Jumbotron>
    );
};

export default AboutPage;
