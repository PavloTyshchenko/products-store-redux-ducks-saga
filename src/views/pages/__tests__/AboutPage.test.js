import React from 'react';
import { shallow } from 'enzyme';

import AboutPage from '../AboutPage';


describe('AboutPage Component', () => {

    it('Should render without errors', () => {
        const component = shallow(<AboutPage />);
        expect(component.length).toBe(1);
    });

    test('Should match snapshot', () => {
        const component = shallow(<AboutPage />);
        expect(component.debug()).toMatchSnapshot();
    });
});