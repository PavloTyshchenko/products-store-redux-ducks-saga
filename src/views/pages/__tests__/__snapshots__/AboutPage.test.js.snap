// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`AboutPage Component Should match snapshot 1`] = `
"<Bootstrap(Jumbotron)>
  <h1>
    About
  </h1>
  <p>
    This is a simple react application to test myself, practice skills and obtain some new knowledge
  </p>
  <p>
    <Button variant=\\"primary\\" active={false} disabled={false} type=\\"button\\">
      Learn more
    </Button>
  </p>
</Bootstrap(Jumbotron)>"
`;
